buildscripts/github/llvmlite_evaluate.py

```diff
@@ -29,6 +29,12 @@
             "python-version": python_version,
         })
 
+# Exclude Python 3.14 for osx-64
```

Comment:
This is a bit of anti-pattern. We don't really want this kind of code here. Instead we want a declarative approach. All the previous instantiations of a Numba CI system had this anti-pattern and it just caused continuous headaches.